azurerm/resource_arm_monitor_action_group_test.go

```diff
@@ -440,41 +954,319 @@ resource "azurerm_monitor_action_group" "test" {
-  }
+	publish_content_link {
+		uri = "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-automation-runbook-getvms/Runbooks/Get-AzureVMTutorial.ps1"
+	}
+}
+`, rInt, location, rInt, rInt)
+}
 
-  webhook_receiver {
-    name        = "callmybackupapi"
-    service_uri = "https://backup.example.com/warning"
-  }
+func testAccAzureRMMonitorActionGroup_voiceReceiver(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_monitor_action_group" "test" {
+  name                = "acctestActionGroup-%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  short_name          = "acctestag"
+
+	voice_receiver {
+		name         = "oncallmsg"
+		country_code = "1"
+		phone_number = "1231231234"
+	}
 }
 `, rInt, location, rInt)
 }
 
+func testAccAzureRMMonitorActionGroup_logicAppReceiver(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_monitor_action_group" "test" {
+  name                = "acctestActionGroup-%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+	short_name          = "acctestag"
+
+	logic_app_receiver {
+		name = "logicappaction"
+		resource_id = "${azurerm_logic_app_workflow.test.id}"
+		callback_url = "http://test-host:100/workflows/fb9c8d79b15f41ce9b12861862f43546/versions/08587100027316071865/triggers/manualTrigger/paths/invoke?api-version=2015-08-01-preview&sp=%%2Fversions%%2F08587100027316071865%%2Ftriggers%%2FmanualTrigger%%2Frun&sv=1.0&sig=IxEQ_ygZf6WNEQCbjV0Vs6p6Y4DyNEJVAa86U5B4xhk"
+	}
+}
+
+resource "azurerm_logic_app_workflow" "test" {
+	name                = "acctestLA-%d"
+	location            = "${azurerm_resource_group.test.location}"
+	resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_logic_app_trigger_http_request" "test" {
+	name         = "some-http-trigger"
+	logic_app_id = "${azurerm_logic_app_workflow.test.id}"
+
+schema = <<SCHEMA
+{
+	"type": "object",
+	"properties": {
+		"hello": {
+			"type": "string"
+		}
+	}
+}
+SCHEMA
+}
+`, rInt, location, rInt, rInt)
+}
+
+func testAccAzureRMMonitorActionGroup_azureFunctionReceiver(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_monitor_action_group" "test" {
+  name                = "acctestActionGroup-%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  short_name          = "acctestag"
+
+	azure_function_receiver {
+		name = "funcaction"
+		function_app_resource_id = "${azurerm_function_app.test.id}"
+		function_name = "myfunc"
+		http_trigger_url = "https://example.com/trigger"
+	}
+}
+
+resource "azurerm_storage_account" "test" {
+	name                     = "acctestSA-%d"
```

Comment:
(as above) we need to make this:

```suggestion
	name                     = "acctestsa%s"
```